cli/tools/bench/reporters.rs

```diff
@@ -178,21 +178,14 @@ impl BenchReporter for ConsoleReporter {
       }
 
       Some(group) => {
-        if self.group.is_none()
-          && self.has_ungrouped
-          && self.group_measurements.is_empty()
-        {
-          println!();
-        }
-
         if self.group.is_none() || group != self.group.as_ref().unwrap() {
           self.report_group_summary();
         }
 
         if (self.group.is_none() && self.has_ungrouped)
           || (self.group.is_some() && self.group_measurements.is_empty())
         {
-          println!();
+          println!("group {}", colors::green(group));
```

Comment:
Slight nitpick, but `group` is not super relevant, so maybe print it using `colors::gray` so it shows as less significant that the actual bench name

Reply:
Done